### PR TITLE
added ROCm Core and AMD SMI

### PR DESCRIPTION
**Pull request by randyh62**
add ROCm-Core
add AMD SMI
restore paragraph on ROCm packages

### CODE REVIEW

File: docs/about/license.md [hunk 1]
```diff
@@ -85,8 +87,8 @@ additional licenses. Please review individual repositories for more information.
 | hsa-amd-aqlprofile | [AMD Software EULA](https://www.amd.com/en/legal/eula/amd-software-eula.html)
 
 Open sourced ROCm components are released via public GitHub
-repositories, packages on `https://github.com/ROCm` and other distribution channels.
-Proprietary products are only available with special permission from AMD. Currently, only
+repositories, packages on `https://repo.radeon.com` and other distribution channels.
```
Comment (yhuiYH) [inline docs/about/license.md:90]: The URLs should be links, not just codetags.  Is this the MD syntax?

```suggestion
repositories, packages on [https://repo.radeon.com](https://repo.radeon.com) and other distribution channels.
```

File: docs/about/license.md [hunk 2]
```diff
@@ -85,8 +87,8 @@ additional licenses. Please review individual repositories for more information.
...
+Proprietary products are only available on `https://repo.radeon.com`. Currently, only
```
Comment (yhuiYH) [inline docs/about/license.md:91]:
```suggestion
Proprietary products are only available on [https://repo.radeon.com](https://repo.radeon.com). Currently, only
```
Same...

**Review by yhuiYH — DISMISSED** [review]
LGTM. Merge when ready

File: docs/about/license.md (hunk 1 above)
Comment (randyh62) [reply docs/about/license.md]: I updated it.

**Review by yhuiYH — APPROVED** [review]
LGTM. Merge when ready.

**Pull request merged**